chains/mainnet/synternet.json

```diff
@@ -0,0 +1,30 @@
+{
+  "chain_name": "synternet",
+  "registry_name": "synternet",
+  "api": [
+      {
+          "address": "https://api.synternet.com", 
+          "provider": "Synternet"
+      }
+  ], 
+  "rpc": [
+      {
+          "address": "https://rpc.synternet.com:443", 
+          "provider": "Synternet"
+      }
+  ],
+  "snapshot_provider": "https://polkachu.com/tendermint_snapshots/synternet",
+  "sdk_version": "0.47.11",
+  "coin_type": "118",
+  "min_tx_fee": "800",
+  "assets": [{
+      "base": "usynt",
+      "symbol": "SYNT",
+      "exponent": "6",
+      "coingecko_id": "synternet-synt", 
+      "logo": "/logos/synternet.png"
+  }],
+  "addr_prefix": "synt",
+  "theme_color": "#272d45",
```

Comment:
Why this color? I probably don't know something.

Reply:
I used the same color as specified [here](https://github.com/cosmos/chain-registry/blob/master/synternet/chain.json#L170).
Although... It says "primary color" so might be not what we need for "there_color".

Reply:
Do we have another theme color? Or we can go with this one?

Reply:
i don't know what that is, probably not important, so let's go with this one